Yoast/wordpress-seo
# Use get analysis data inside new refresh

## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

* Ussed the data from `getAnalysisData` inside the new refresh.

## Test instructions

This PR can be tested by following these steps:

* Open a new post/category page in the classic editor or Gutenberg and check whether the values in the paper are correct.

## Quality assurance


Fixes #https://github.com/Yoast/YoastSEO.js/issues/1702
